nemo/collections/llm/recipes/gemma_7b.py

```diff
@@ -277,9 +275,6 @@ def finetune_recipe(
         on fine-tuning LLMs with NeMo, see the fine-tuning guide in the
         `examples/llm/finetune/` directory.
     """
-    # Disable cuDNN attention since TE 1.8 does not support head dim > 128
-    os.environ['NVTE_FUSED_ATTN'] = "0"
```

Comment:
if you remove this, does the recipe still run?

Reply:
It's now being handled at the mcore level I believe

Reply:
@cuichenx see the corresponding change to `nemo/collections/llm/gpt/model/gemma.py` - the `attention_backend` setting now sets these environment variables for users, so it's an explicit opt-in of what attention implementation to use, instead of relying on TE's fallback approach

Reply:
ah okay, thanks!